src/utils/mocks/index.ts

```diff
@@ -17,7 +17,7 @@ const KFI: IAsset = {
     Wallet: 'https://klever.io/',
     Website: 'https://klever.org/',
     Whitepaper:
-      'https://klever.finance/wp-content/uploads/2021/10/Klever_Finance_Whitepaper_v1.1.pdf',
+      'https://klever.org/wp-content/uploads/2021/10/Klever_Finance_Whitepaper_v1.1.pdf',
```

Comment:
link don't work

Reply:
but i think i'ts pecause the wapper is on this link https://storage.googleapis.com/kleverchain-public/Klever-Blockchain-Whitepaper-v.2.0-lr.pdf